boards.txt

```diff
@@ -25574,6 +25574,537 @@ heltec_ht_de01.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_vision_master_e290.name=Heltec Vision Master E290
+heltec_vision_master_e290.vid.0=0x303a
+heltec_vision_master_e290.pid.0=0x1001
```

Comment:
please remove VID/PID from all board definitions. This is a generic VID/PID of the CDC/JTAG and should exist only once in this file.

Reply:
Hello, I have removed VID and PID, please continue to review.